content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+    option to skip IAP**. IAP ensures you can connect securely to the Kubeflow
+    web applications.
+  * When setting up the **authorized redirect URI** for the **OAuth client 
+    credentials**, use the same value for the `<deployment_name>` as you used
+    when setting up the `DEPLOYMENT_NAME` environment variable earlier in this
+    tutorial.
+  * Use the Kubeflow **deployment UI** as a quick way to set up a Kubeflow 
+    deployment on GCP. The getting-started guide describes how to use the
+    deployment UI. If you want more control over the configuration of your
+    deployment you can use the `kfctl.sh` script instead of the UI. The script
+    is also described in the getting-started guide.
+  * Choose **Kubeflow version v0.4.1** or later.
+
+    The following screenshot shows the Kubeflow deployment UI with hints about
+    the value for each input field:
+
+    <img src="/docs/images/gcp-e2e-deploy-kubeflow.png" 
+        alt="Prediction UI"
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. (Optional) Use `kubectl` to connect to your cluster:
+
+  * Connect your local `kubectl` session to the cluster:
+
+      ```
+      gcloud container clusters get-credentials \
+          ${DEPLOYMENT_NAME} --zone ${ZONE} --project ${PROJECT}
+      ```
+
+  * Switch to the `kubeflow` namespace to see the resources on the Kubeflow 
+    cluster:
+
+      ```
+      kubectl config set-context $(kubectl config current-context) --namespace=kubeflow
+      ```
+
+  * Check the resources deployed in the `kubeflow` namespace:
+
+      ```
+      kubectl get all
+      ```
+
+1. Access the Kubeflow UI, which becomes available at the following URI after 
+   several minutes:
+
+    ```
+    https://<deployment-name>.endpoints.<project>.cloud.goog/
+    ```
+
+    The following screenshot shows the Kubeflow UI:
+    <img src="/docs/images/central-ui.png" 
+        alt="Prediction UI"
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click **Pipeline Dashboard** to access the pipelines UI. The pipelines UI
+   looks like this:
+    <img src="/docs/images/pipelines-ui.png" 
+      alt="Pipelines UI"
+      class="mt-3 mb-3 border border-info rounded">
+
+Notes:
+
+* When the deployment has finished, you should have a running cluster in the 
+  cloud ready to run your code. You can interact with the cluster either by 
+  using [`kubectl`][kubectl] or by going to the
+  [GKE page on the GCP Console][gcp-console-kubernetes-engine].
+
+* While the deployment is running, you can watch your resources appear on the
+  GCP console:
+    * [Deployment on Deployment Manager][gcp-console-deployment-manager]
+    * [Cluster on GKE][gcp-console-kubernetes-engine]
+    * [Computing resources on Compute Engine][gcp-console-compute-engine]
+
+* It can take 10-15 minutes for the URI to become available. Kubeflow needs
+  to provision a signed SSL certificate and register a DNS name.
```

Comment:
This information might be more useful at step 3 above, when the reader is first waiting for the UI to appear. Gives them something to read and sets expectations

Reply:
I did debate putting it there, put it seemed like too much info and therefore more useful in the notes section. Step 3 says that the Kubeflow UI "becomes available at the following URI after several minutes...".